From 2a46b0c868842967f7704bb4dda978445bd24f99 Mon Sep 17 00:00:00 2001
From: Han Xu <b45815@freescale.com>
Date: Thu, 4 Jun 2015 11:47:21 -0500
Subject: [PATCH] MLK-11043: ARM: dts: add per1_bch clock for NAND on i.MX6UL

Bch use AXI bus transfer data through fabric, need to enable this clock
for bch at fabric side.

Signed-off-by: Han Xu <b45815@freescale.com>
---
 arch/arm/boot/dts/imx6ul.dtsi | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 84bf805..35cb6ed 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -1115,9 +1115,10 @@
 			clocks = <&clks IMX6UL_CLK_GPMI_IO>,
 				 <&clks IMX6UL_CLK_GPMI_APB>,
 				 <&clks IMX6UL_CLK_GPMI_BCH>,
-				 <&clks IMX6UL_CLK_GPMI_BCH_APB>;
+				 <&clks IMX6UL_CLK_GPMI_BCH_APB>,
+				 <&clks IMX6UL_CLK_PER_BCH>;
 			clock-names = "gpmi_io", "gpmi_apb", "gpmi_bch",
-				      "gpmi_bch_apb";
+				      "gpmi_bch_apb", "per1_bch";
 			dmas = <&dma_apbh 0>;
 			dma-names = "rx-tx";
 			status = "disabled";
-- 
1.8.0

